### DISCUSSION TITLE
mo.ui.table clicking anywhere in a pre-selected row (including highlighting) un-selects the row

### DISCUSSION THREAD

**Discussion by delennc** [marimo-team/marimo#1720] [General]
### Describe the bug

Selecting a row in mo.ui.table requires clicking checkbox, but clicking anywhere in the row un-selects it (ideally un-selection would only happen when clicking checkbox)

### Environment

0.6.22

**Comment by mscolnick** [comment]
this is actually by design. there are a few other UX that we looked into that support this and find it quite useful. going to move this to a discussion before making it an issue that we would like to change

**Reply by mscolnick** [reply]
something we could consider (assuming you are copying the text in the row) is detecting drag and not selecting
